presto-memsql/src/test/java/io/prestosql/plugin/memsql/TestMemSqlTypeMapping.java

```diff
@@ -517,18 +563,24 @@ private DataSetup prestoCreateAsSelect(Session session, String tableNamePrefix)
         return new CreateAsSelectDataSetup(new PrestoSqlExecutor(getQueryRunner(), session), tableNamePrefix);
     }
 
+    private DataSetup prestoCreateAndInsert(Session session, String tableNamePrefix)
+    {
+        return new CreateAndInsertDataSetup(new PrestoSqlExecutor(getQueryRunner(), session), tableNamePrefix);
+    }
+
     private DataSetup memSqlCreateAndInsert(String tableNamePrefix)
     {
         return new CreateAndInsertDataSetup(memSqlServer::execute, tableNamePrefix);
     }
 
-    private static DataType<String> jsonDataType(Function<String, String> toLiteral)
+    private static DataType<LocalDate> memSqlDateDataType(Function<LocalDate, String> toLiteral)
+    {
+        return dataType("date", DATE, toLiteral, identity());
+    }
+
+    private static DataType<String> memSqlJsonDataType(Function<String, String> toLiteral)
```

Comment:
please do the rename jsonDataType -> memSqlJsonDataType
in the commit which intro'd the function